modules/dev-utils/src/main/java/org/apache/ignite/development/utils/indexreader/IgniteIndexReader.java

```diff
@@ -491,6 +503,20 @@ public void readIdx() {
         }
     }
 
+    /** */
+    private void printFileReadingErrors(List<Throwable> idxStoreErrors, Map<Long, List<Throwable>> partStoresErrors) {
```

Comment:
It is enough to display only the map, but with a check whether the partition is index.